enhancements/sig-policy/78-policy-collection/README.md

```diff
@@ -0,0 +1,77 @@
+# Policy Collection
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in
+      [website](https://github.com/open-cluster-management/website/)
+
+## Summary
+
+The goal of this feature is to automate and simplify deployment of policies, policy sets and
+policy generator projects by creating (transferring) a community that provides best practices, samples, blogs
```

Comment:
need some clarification on the meaning of community here. does it mean to be ocm community?

Reply:
I tried to clarify this.  Please review again.